contracts/contracts/strategies/ConvexStrategy.sol

```diff
@@ -18,20 +18,8 @@ contract ConvexStrategy is BaseCurveStrategy {
     using StableMath for uint256;
     using SafeERC20 for IERC20;
 
-    event RewardTokenCollected(
-        address recipient,
-        address token,
-        uint256 amount
-    );
-
-    event CvxRewardTokenAddressUpdated(
-        address _oldAddress,
-        address _newAddress
-    );
-
     address internal cvxDepositorAddress;
     address internal cvxRewardStakerAddress;
-    address public cvxRewardTokenAddress;
```

Comment:
Let's just mark this as deprecated, rather than dropping this storage slot.

We are setting cvxDepositorPTokenId in the initializer, which is called during the deploy, so the all storage slots we use will be correct, and this will work as is for now. However we leave a the next unused storage slot now containing a value. This has a tiny chance of tripping us up at some future time, if we expected it be just like any any other empty storage slot.

Reply:
That's a good point... fixed. I've seen this when testing mainnet fork without deprecating slots and replacing previously used slot of type `address`, with an `address[]` type. Can't remember if the node hanged or has thrown a weird exception. 

I think slot was initialised with an `address` type and then when replaced with `address[]` it was expecting list length in that slot but got something completely different.

Reply:
Haha, yeah, getting an address when it was expecting a length could make it thing it was an insanely long list.